app/src/main/java/com/infomaniak/mail/ui/newMessage/NewMessageViewModel.kt

```diff
@@ -618,15 +615,16 @@ class NewMessageViewModel @Inject constructor(
         isFinishing: Boolean,
         isTaskRoot: Boolean,
         rawSubject: String,
+        rawUiBody: String,
```

Comment:
calling it raw is sort of implying that it will have some computing on the value, which is not the case here, so i'm not a big fan